libraries/AP_Scripting/examples/orbit_follow.lua

```diff
@@ -0,0 +1,49 @@
+-- example that does an orbit of a vehicle that is being followed
+-- by adjusting the follow offset X and Y parameters
+-- user settable FOLL_ORB_TIME and FOLL_ORB_RADIUS parameters
+-- are provided for the time for one orbit (in seconds) and the radius
+-- of the orbit
+
+-- setup param block for FOLL_ extensions
+local PARAM_TABLE_KEY = 83
+assert(param:add_table(PARAM_TABLE_KEY, "FOLL_", 2), 'could not add param table')
+
+function bind_param(name)
+   local p = Parameter()
+   assert(p:init(name), string.format('could not find %s parameter', name))
+   return p
+end
+
+function bind_add_param(name, index, default_value)
+   assert(param:add_param(PARAM_TABLE_KEY, index,  name, default_value), string.format('could not add %s', name))
+   return bind_param(name)
+end
+
+local FOLL_OFS_X = bind_param("FOLL_OFS_X")
```

Comment:
I like the parameter names and in fact these are the same that I chose for the AP_Follow library.  We probably need to be careful of conflicts if used with Copter or Rover.  Maybe the comments at the top should specify that this script is only for Plane and/or perhaps we should name the script starting with "plane-".  I've prefixed the copter only scripts with "copter-".